src/metabase/db/custom_migrations.clj

```diff
@@ -23,11 +23,23 @@
    [toucan2.core :as t2]
    [toucan2.execute :as t2.execute])
   (:import
+   (liquibase Scope)
+   (liquibase.change Change)
    (liquibase.change.custom CustomTaskChange CustomTaskRollback)
-   (liquibase.exception ValidationErrors)))
+   (liquibase.exception ValidationErrors)
+   (liquibase.util BooleanUtil)))
 
 (set! *warn-on-reflection* true)
 
+(defn should-execute-change?
+  "Check if the change is supposed to be executed.
+  This is a work around. The rollback method is called twice: once
+  for generating MDC data and once for actually making the change.
+  The same problem has been fixed for forward changes in Liquibase
```

Comment:
curious: where does the problem get fixed for forward change?

Reply:
liquibase/liquibase#4054